src/screens/WalletScreen/index.js

```diff
@@ -49,35 +50,61 @@ type Actions = {
    * @desc Function to request wallet list update.
    */
   updateWalletList: () => void,
-}
+};
 
 type State = {
   /**
    * @desc Flag to control Refreshing on Pull to Refresh
    */
   pullToRefreshInProgress: boolean,
-}
+  /**
+   * @desc Flag to control opening of transactions Modal
+   */
+  transactionsVisible: boolean,
+  /**
+   * @desc To store transaction variables
```

Comment:
Naming and description of the field are not clear at least for me. Why the name `model`? I guess you wanted to use `modal`?

Reply:
Done